### PR TITLE
Update Set-CsTeamsMessagingPolicy.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**
per discussion with PM John Taig

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md [hunk 1]
```diff
@@ -204,6 +204,7 @@ Accept wildcard characters: False
 
 ### -AllowUserDeleteMessage
 Determines whether a user is allowed to delete their own messages. Set this to TRUE to allow. Set this to FALSE to prohibit.
+if this value is set to false the teams owner will not be able to delete his own messages
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md:207]:
```suggestion
If this value is set to FALSE, the team owner will not be able to delete his own messages.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RuiPereiraTabares, kindly commit the suggested changes. Many thanks.

**Comment by RuiPereiraTabares** [comment]
@JohanFreelancer9  done! many thanks

**Review by JohanFreelancer9 — APPROVED** [review]

File: skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md [hunk 2]
```diff
@@ -204,6 +204,7 @@ Accept wildcard characters: False
 
 ### -AllowUserDeleteMessage
 Determines whether a user is allowed to delete their own messages. Set this to TRUE to allow. Set this to FALSE to prohibit.
+If this value is set to FALSE, the team owner will not be able to delete his own messages.
```
Comment (get-itips) [inline skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md:207]: So even if AllowOwnerDeleteMessage is true? Thanks for clarifying

Comment (RuiPereiraTabares) [reply skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md:207]: even if allowownerdeletemessage is on true the teams owner will not be able to delete the messages his sent in a team channel

**Review by dariomws — COMMENTED** [review]
Similar to https://github.com/MicrosoftDocs/office-docs-powershell/pull/9152, I would add this sentence under the AllowOwnerDeleteMessage description.

**Comment by yogkumgit** [comment]
@RuiPereiraTabares Please check the review comments and update. Thanks

File: skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md (hunk 2 above)
Comment (tseward) [inline skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md:207]: Instead of using 'his', can you use 'their' to be gender-neutral?

**Review by tseward — CHANGES_REQUESTED** [review]

File: skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md (hunk 2 above)
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md:207]:
```suggestion
If this value is set to FALSE, the team owner will not be able to delete their own messages.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@RuiPereiraTabares, could you please commit the suggested change?

**Comment by RuiPereiraTabares** [comment]
@JohanFreelancer9 approved commint

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips @dariomws

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**